sqlutil/sqlutil.go

```diff
@@ -73,3 +73,99 @@ func (l *StringList) Scan(v interface{}) error {
 	*l = strs
 	return nil
 }
+
+// Bool add the capability to the sql driver to work directly with bool types.
+type Bool bool
```

Comment:
Maybe it would be a good idea to add some documentation about what formats this supports and why it improves over the native `bool` parsing? That is not immediately obvious from just reading the documentation right now.

Reply:
Sure! I will improve that. 😄

Reply:
✅ Done